openai/requirements.txt

```diff
@@ -0,0 +1 @@
+openai>=0.27.0,<1.0
```

Comment:
Ah, I see now what's going on.
https://pypi.org/project/openai/ Requires minimum `Python >=3.7.1`, while the tests are ran under the python 3.6

The technical debt of maintaining the Python versions hit us everywhere: https://github.com/StackStorm/community/issues/103

Reply:
I changed the workflow to use 3.8 and it looks like it's passing now, though I'm not sure if I did it in a way that it would be kept if it were to be moved to the Exchange.